toolchain/check/import_ref.cpp

```diff
@@ -2003,18 +2003,18 @@ static auto TryResolveTypedInst(ImportRefResolver& resolver,
     return ResolveResult::Retry();
   }
 
-  for (auto& vtable_entry_id : lazy_virtual_functions) {
+  for (auto p : llvm::zip(virtual_functions, lazy_virtual_functions)) {
```

Comment:
Can you use a structured binding here instead of using `get<i>` below? I think that'd be more readable.

Reply:
Oh, right, sorry, got a bit jumbled up thinking about a tuple with a reference member in a structured binding and how that'd all work.

Yeah, that's better - in 728774362c3c4b9d5bd93e672b9a8abb7ce8eadc